src/mantle/support/traits/trait-hookable.php

```diff
@@ -247,4 +268,26 @@ protected function fire_doing_it_wrong( ReflectionMethod $method ): void {
 
 		_doing_it_wrong( esc_html( static::class . '::' . $method->getName() ), esc_html( $message ), '1.0.0' );
 	}
+
+	/**
+	 * Fire all validators for a method.
+	 *
+	 * @param ReflectionMethod $method The hook callback method.
+	 * @return bool
+	 */
+	protected function fire_validator( ReflectionMethod $method ): bool {
+		$attributes = $method->getAttributes();
+
+		// Check all validators for this method.
+		foreach ( $attributes as $attribute ) {
+			$instance = $attribute->newInstance();
+			if ( $instance instanceof \Mantle\Types\Validator ) {
```

Comment:
We have other validators at alley. Not sure if we want to expand this. But I used the one the framework supports.